src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,23 @@
+// @import '../utils/variables.scss';
+
+.catalog {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  padding: $paddingY $paddingX;
+  width: 4 * $cardWhidth + 3 * $gapX + 2 * $paddingX;
```

Comment:
You can't set width and margin for BEM block

Reply:
Agree, fix this comment